microcosm_flask/tests/conventions/test_command.py

```diff
@@ -0,0 +1,131 @@
+"""
+Command operation tests.
+
+"""
+from json import dumps, loads
+from hamcrest import (
+    assert_that,
+    equal_to,
+    is_,
+)
+
+from marshmallow import fields, Schema
+from microcosm.api import create_object_graph
+
+from microcosm_flask.conventions.encoding import dump_response_data, load_request_data
+from microcosm_flask.conventions.registry import request, response
+from microcosm_flask.conventions.swagger import register_swagger_endpoint
+from microcosm_flask.operations import Operation
+
+
+class CommandArgumentSchema(Schema):
+    value = fields.String(required=True)
+
+
+class CommandResultSchema(Schema):
+    result = fields.Boolean(required=True)
+    value = fields.String(required=True)
+
+
+def make_command(graph, request_schema, response_schema):
+    """
+    Create an example command route.
+
+    """
+
+    @graph.route("/v1/foo/do", Operation.Command, "foo")
```

Comment:
What do you think about name spacing non-RESTful actions? The idea could be similar to https://github.com/interagent/http-api-design/blob/503b92f9db25f23f66bc512be86aac6f562c2427/en/requests/actions.md#L1